### ISSUE TITLE
Add support for Apple Pencil scribble

### ISSUE THREAD

**Issue by sibartlett** [excalidraw/excalidraw#3091]
As a user with an iPad and Apple Pencil,
I want to use [Scribble](https://support.apple.com/en-ca/guide/ipad/ipad355ab2a7/ipados),
to enter text without using the keyboard.

**Comment by dwelle** [comment]
Hard to debug this without an iPad Pencil. Not sure how it'd work tho, because I believe you'd still need to enter the text editor (`<textarea>`) — which should actually work even now if you create a text shape and *then* use the pencil (tho the empty textarea is basically 0px wide, so probably not until you entered some text first).

**Comment by sibartlett** [comment]
Yeah, I think the problem is that the texture is basically 0px wide.

I'll try the workaround suggested, but it's not ideal. So I'll also try to have a stab at debugging the problem, when I get a chance.
